src/custom/components/Tokens/TokensTableRow.tsx

```diff
@@ -24,13 +35,107 @@ type DataRowParams = {
   balance?: CurrencyAmount<Token> | undefined
   handleBuy: (token: Token) => void
   handleSell: (token: Token) => void
+  closeModal: () => void
```

Comment:
not related to this PR, but given that handleBuy and sell are SO similar, I would probably unify the functions and pass as a parameter if it is a buy or a sell.

It might reduce the prop drilling and also will refactor pieces of code that are very similar

```ts
  const handleBuy = useCallback(
    (token: Token) => {
      history.push(`/swap?outputCurrency=${token.address}`)
    },
    [history]
  )

  const handleSell = useCallback(
    (token: Token) => {
      history.push(`/swap?inputCurrency=${token.address}`)
    },
    [history]
  )
  ```

Reply:
Fixed!